uiplib/settings.py

```diff
@@ -1,7 +1,7 @@
 import os
 import json, argparse
 
-HOME_DIR = os.path.join(os.path.expanduser("~"), '.uip')
+HOME_DIR = os.path.join(os.path.expanduser("~"), 'uip-pics')
```

Comment:
this is the complete home directory right? it should be named `uip` instead of `uip-pics` right? it would also have the `settings.json` also does this make it work for you?